target_type: 'aprilgrid' #gridtype
tagCols: 7               #number of apriltags CAREFUL: ROW/COLS must match how the tag is generated!!!
tagRows: 5               #number of apriltags
tagSize: 0.04         #size of apriltag, edge to edge [m]
tagSpacing: 0.25        #ratio of space between tags to tagSize
